### DISCUSSION TITLE
Decona for haploid organisms

### DISCUSSION THREAD

**Discussion by leocaserta** [Saskia-Oosterbroek/decona#18] [Ideas]
Hi, 
Does Decona work for haploid genomes? Since Medaka has the tool "Medaka haploid variant", I was wondering if it could be implemented as an option when running Decona because the regular Medaka variant tool generates many spurious indels when used for viruses, for example.

Thank you, 

Leonardo

**Comment by Saskia-Oosterbroek** [comment]
Hi Leonardo,
This is not something Decona was intended for, so the variant option in Decona is “medaka_vatiant”. I think it should be relatively easy for you to replace that command with the haploid command in the Decona script for your own use :)

Best,
Saskia
